MTM-53127 Added icon selector info to the docs. (#1459)
SoftwareAG/c8y-docs: content/standard-tenant/ecosystem-bundle/custom-applications.md

1. Click **Add application** at the top right of the **Applications** tab.
2. In the upcoming dialog, select **Duplicate existing application**.
3. Select the desired application from the dropdown list, for example "Cockpit".

> 4. In the next window, provide a name for the application, an application key to identify the application, a path as part of the URL to invoke the application and finally select an icon for the new applicaiton via the select icon modal. Per default, the values of the original application are provided, extended by a number. If you set the path to the path of the original subscribed application, your own application will overrule the subscribed application.

Reviewer: ```suggestion
4. In the next window, provide a name for the application, an application key to identify the application, and a path as part of the URL to invoke the application.  Finally select an icon for the new application from the available icons. Per default, the values of the original application are provided, extended by a number. If you set the path to the path of the original subscribed application, your own application will overrule the subscribed application.
```
